packages/slice-machine/lib/env/index.ts

```diff
@@ -111,7 +111,7 @@ function parseStorybookConfiguration(cwd: string) {
 export async function getEnv(
   maybeCustomCwd?: string
 ): Promise<{ errors?: { [errorKey: string]: ServerError }; env: Environment }> {
-  const cwd = maybeCustomCwd || process.env.CWD;
+  const cwd = maybeCustomCwd || process.env.CWD || process.cwd();
```

Comment:
In which case do you need that last option ?

Reply:
in one of the tests